### PR TITLE
Moving proposals to accepted (LFX Mentorship 2023 term 1: March - May)

### PR DESCRIPTION
**Pull request by nate-double-u**
Draft till approved by LFX.

* Adding Harbor projects

### CODE REVIEW

File: lfx-mentorship/2023/01-Mar-May/README.md [hunk 1]
```diff
@@ -183,6 +188,45 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
 - Upstream Issue: https://github.com/cortexproject/cortex/issues/5107
 - LFX URL: https://mentorship.lfx.linuxfoundation.org/project/0071e2ff-f538-4817-978b-07b267cfcd6a
 
+#### Harbor
```
Comment (thisisobate) [inline lfx-mentorship/2023/01-Mar-May/README.md:191]:
```suggestion
### Harbor
```

File: lfx-mentorship/2023/01-Mar-May/README.md [hunk 2]
```diff
@@ -14,6 +14,11 @@
       - [API to import Prometheus \& Thanos blocks](#api-to-import-prometheus--thanos-blocks)
       - [Switch Cortex Ruler to query Query Frontend](#switch-cortex-ruler-to-query-query-frontend)
       - [Automated nightly benchmarks](#automated-nightly-benchmarks)
+    - [Harbor](#harbor)
+      - [Regex replication rules](#regex-replication-rules)
+      - [An official Golang API client and CLI for Harbor](#an-official-Golang-API-client-and-CLI-for-Harbor)
```
Comment (thisisobate) [inline lfx-mentorship/2023/01-Mar-May/README.md:19]:
```suggestion
      - [An official Golang API client and CLI for Harbor](#an-official-golang-api-client-and-cli-for-harbor)
```

**Review by thisisobate — CHANGES_REQUESTED** [review]

File: lfx-mentorship/2023/01-Mar-May/README.md [hunk 3]
```diff
@@ -183,6 +188,45 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
...
+##### An official Golang API client and CLI for Harbor
+
+- Description: Design, plan and implement an Golang API client for Harbor
+- Expected Outcome: Working golang harbor API client which can be used in the CI/CD implementations which compliments the Web UI, well documented and with the coresponding architectural diagrams under the Harbor org(not necessary to be complete functionality)
+- Recommended Skills: Golang, spf13/cobra
+- Mentor(s): Vadim Bauer, vb@container-registry.com); @wy65701436 - Yan Wang(wangyan@vmware.com);
+- Upstream Issue (URL): https://github.com/search?q=Harbor%20CLI&type=repositories
+- LFX URL: https://mentorship.lfx.linuxfoundation.org/project/7e8cb88a-5b37-471c-8db8-e11907b5a661
+
+##### Implement per project and/or for the whole instance vulnerability overview
```
Comment (thisisobate) [inline lfx-mentorship/2023/01-Mar-May/README.md:212]:
```suggestion
#### Implement per project and/or for the whole instance vulnerability overview
```

File: lfx-mentorship/2023/01-Mar-May/README.md [hunk 4]
```diff
@@ -183,6 +188,45 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
 - Upstream Issue: https://github.com/cortexproject/cortex/issues/5107
 - LFX URL: https://mentorship.lfx.linuxfoundation.org/project/0071e2ff-f538-4817-978b-07b267cfcd6a
 
+### Harbor
+
+##### Regex replication rules
```
Comment (thisisobate) [inline lfx-mentorship/2023/01-Mar-May/README.md:193]:
```suggestion
#### Regex replication rules
```

File: lfx-mentorship/2023/01-Mar-May/README.md [hunk 5]
```diff
@@ -183,6 +188,45 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
...
+  
+- Description: Design, plan and implement an and UI and backend to be able to visualize per project and/or for the registry vulnerability overview which will allow better security audits and vulenrability mitigation 
+- Expected Outcome: Addition to the Web UI which can be used to represent in full for the whole Harbor instance or per project the vulnerability status of the images, which will allow Harbor admin or project admin to get an overview of the existing vulnerabilities on in the images, also to provide capability to export the data via the CVE exporter so it can be consumed in 3rd party tools(not necessary to be complete functionality)
+- Recommended Skills: Angular, JavaScript, Golang, UI/UX, Clarity 
+- Mentor(s): Vadim Bauer, vb@container-registry.com); @wy65701436 - Yan Wang(wangyan@vmware.com);
+- Upstream Issue (URL): https://github.com/goharbor/harbor/issues/16680 https://github.com/goharbor/harbor/issues/10496 https://dso.docker.com/explore?search=pkgs
+- LFX URL: https://mentorship.lfx.linuxfoundation.org/project/7ea4c506-c830-4a15-be4a-600d2dfe3f44
+
+##### Harbor Robot accounts with full Harbor API access
```
Comment (thisisobate) [inline lfx-mentorship/2023/01-Mar-May/README.md:221]:
```suggestion
#### Harbor Robot accounts with full Harbor API access
```

File: lfx-mentorship/2023/01-Mar-May/README.md [hunk 6]
```diff
@@ -183,6 +188,45 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
...
+
+- Description: Add more versatile replication filters
+- Expected Outcome: Implement regex capability when defining relication rules, update documentation and present the functionality
+- Recommended Skills: Angular, JavaScript, Golang
+- Mentor(s): vb@container-registry.com@Vad1mo @wy65701436 @OrlinVasilev
+- Mentor(s): @Vad1mo - Vadim Bauer, vb@container-registry.com); @wy65701436 - Yan Wang(wangyan@vmware.com); @OrlinVasilev Orlin Vasilev (ovasilev@vmware.com)
+- Upstream Issue (URL): https://github.com/goharbor/harbor/issues/8614 
+- LFX URL: https://mentorship.lfx.linuxfoundation.org/project/49749be9-5a67-4b2b-9312-7def13ae98b8
+
+##### An official Golang API client and CLI for Harbor
```
Comment (thisisobate) [inline lfx-mentorship/2023/01-Mar-May/README.md:203]:
```suggestion
#### An official Golang API client and CLI for Harbor
```

**Review by thisisobate — CHANGES_REQUESTED** [review]
We need to increase the heading to correspond with the rest of the document.
Let me know once you update it so I can approve.

**Comment by nate-double-u** [comment]
I'm going to override the branch protections on this PR as it's late Friday or Saturday for a lot of the reviewers, and since LFX has approved these I'd like to make sure they're in the list over the weekend for mentees to find.

Thanks for the updates/suggestions @thisisobate 🙂

**Pull request merged**